README.md

```diff
@@ -121,6 +121,26 @@ You can make skt output junit-compatible results by adding a `--junit
 <JUNIT_DIR>` option to any of the above commands. The results will be written
 to the `<JUNIT_DIR>` directory.
 
+Container
+---------
+
+The skt command and all its requirements, as well as a ready for use kernel
+git tree are available in a container. You can run skt commands agains the
+container environment like this:
+
+    % sudo docker run kerneltest/skt -vv build
+
+If you want to enter into the container and play with skt, for debugging
+purposes, or looking at intermediate files:
+
+    $ sudo docker run -ti --entrypoint=/bin/bash kerneltest/skt
+
+If you change the Dockerfile that defines the container, you can rebuild
+the container:
+
+    $ sudo docker build -t docker.io/kerneltest/skt:latest .
+    $ sudo docker push docker.io/kerneltest/skt:latest
+
```

Comment:
Wait, does that push to the same image for everyone? How does that work without authentication?

Reply:
I can add accounts.

Reply:
I mean it wouldn't work for just somebody outside, would it? If so, then let's put the second line in a separate paragraph, saying this is for developers with appropriate access only.

Reply:
Good point. Clarified in the README.